supervisor/api/middleware/security.py

```diff
@@ -65,22 +64,24 @@
     ROLE_DEFAULT: re.compile(
         r"^(?:"
         r"|/.+/info"
-        r"|/addons"
```

Comment:
This would be breaking for any addon that uses this path with the default role?

Reply:
With `free` info call, that is not in use a longer time. Right @frenck ? (I did that for him before the self slug was burn)

Reply:
oh, right they is because /addons have no info call

Reply:
Hmmm... maybe we should add `/addons/self/info` to the ADDON_API_BYPASS?

Reply:
Self is all ready bypass (all calls)

Reply:
Oh right, good morning ☕️☕️